pandas/core/index.py

```diff
@@ -29,3 +30,5 @@
     FutureWarning,
     stacklevel=2,
 )
+
```

Comment:
why is this needed?

Reply:
Do you mean exporting InvalidIndexError in here?

Reply:
yes

Reply:
I will remove it from here.

Reply:
Removed.